src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -29,22 +30,30 @@ public final class RobotPreferences {
   public static final SN_DoublePreference encoderCountsPerTalonFXRotation = new SN_DoublePreference(
       "encoderCountsPerTalonFXRotation", 2048);
 
+  public static final SN_IntPreference auto = new SN_IntPreference("auto", 0);
+
   public static final class DrivetrainPrefs {
     public static final SN_DoublePreference arcadeSpeed = new SN_DoublePreference("arcadeSpeed", 0.65);
     public static final SN_DoublePreference arcadeTurn = new SN_DoublePreference("arcadeTurn", .5);
     public static final SN_DoublePreference arcadeLowSpeed = new SN_DoublePreference("arcadeLowSpeed", 0.4);
     public static final SN_DoublePreference arcadeHighSpeed = new SN_DoublePreference("arcadeHighSpeed", 1.5);
+    public static final SN_DoublePreference arcadeClosedLoopMaxSpeed = new SN_DoublePreference(
+        "arcadeClosedLoopMaxSpeed", 3);
 
-    public static final SN_DoublePreference driveF = new SN_DoublePreference("driveF", 0);
-    public static final SN_DoublePreference driveP = new SN_DoublePreference("driveP", 1);
+    public static final SN_DoublePreference driveF = new SN_DoublePreference("driveF", 0.045);
+    public static final SN_DoublePreference driveP = new SN_DoublePreference("driveP", 0.1);
     public static final SN_DoublePreference driveI = new SN_DoublePreference("driveI", 0);
     public static final SN_DoublePreference driveD = new SN_DoublePreference("driveD", 0);
 
     public static final SN_DoublePreference driveAllowableClosedLoopErrorInches = new SN_DoublePreference(
         "driveAllowableClosedLoopErrorInches", 2);
     public static final SN_DoublePreference driveClosedLoopPeakOutput = new SN_DoublePreference(
         "driveClosedLoopPeakOutput", 1);
+    public static final SN_DoublePreference driveClosedLoopRamp = new SN_DoublePreference("driveClosedLoopRamp", 0);
     public static final SN_DoublePreference driveLoopsToFinish = new SN_DoublePreference("driveLoopsToFinish", 25);
+    public static final SN_DoublePreference driveWheelCircumference = new SN_DoublePreference("driveWheelCircumference",
+        4 * Math.PI);
```

Comment:
FYI if you pass a different value via RobotPreferences it would be multiplied by Math.PI

Reply:
That's perfect for what this is for. Though this is basically a constant since I doubt the wheel circumference is going to change.